components/base/kinematicbase/kinematics.go

```diff
@@ -117,6 +132,11 @@ func (ddk *differentialDriveKinematics) GoToInputs(ctx context.Context, desired
 				return nil
 			}
 		}
+
```

Comment:
The above is going to have to change too.  The errors aren't checked (something I forgot to do the first time through) and now we could be skipping updating the current Position since we have the possibility to continue)